Makefile

```diff
@@ -20,6 +20,13 @@ release: linux-plugins
 	docker system prune -f
 	docker build -t amazon/aws-for-fluent-bit:init-latest -f Dockerfile.init .
 
+.PHONY: local
+local: linux-plugins
+	docker system prune -f
+	docker build --build-arg LOCAL_BUILD=true -t amazon/aws-for-fluent-bit:latest -f Dockerfile .
```

Comment:
Do we need to pass  LOCAL_BUILD here? Its not used in the main `Dockerfile` I think

Reply:
Still have same question